app/lang/en/setup.php

```diff
@@ -7,5 +7,7 @@
     'status_page_setup' => 'Status Page Setup',
     'show_support'      => 'Show support for Cachet? <small>Places a link in the footer linking to Cachet.</small>',
     'admin_account'     => 'Administrator Account',
-    'complete_setup'    => 'Complete Setup'
+    'complete_setup'    => 'Complete Setup',
+    'completed'         => 'Cachet has been configured successfully!',
```

Comment:
just add the english translation. we do this to keep the files perfectly synced, formatting and all